lib/helpers.js

```diff
@@ -4,7 +4,14 @@
 import {installHandler} from 'web-credential-handler';
 import {Notify} from 'quasar';
 
-export function notifyAllowWallet() {
+const notifiedAccounts = new Map();
+
+export function notifyAllowWallet(account) {
```

Comment:
We should probably make this function have an effect unless the caller wants to prevent from happening if it has already notified.

```suggestion
export function notifyAllowWallet({account, preventRenotify = false}) {
```